architecture/index.adoc

```diff
@@ -146,3 +147,85 @@ automatically delivered into the container. This enables the infrastructure to
 manage secrets for pulling and pushing images, builds, and the deployment
 components, and also allows application code to easily leverage those secrets.
 endif::[]
+
+Communication channels with the REST API as well as between
+xref:infrastructure_components/kubernetes_infrastructure.adoc#master-components[master components]
+such as etcd and API server, are secured with
+link:https://en.wikipedia.org/wiki/Transport_Layer_Security[TLS]. It is
+perhaps better known under its former name "SSL" or by its use in "HTTPS". TLS
+provides strong encryption, data integrity, and authentication of servers with
+X.509 server certificates and public key infrastructure.
+ifdef::openshift-origin,openshift-enterprise,openshift-dedicated[]
```

Comment:
Yes, the custom PKI detail is not relevant for openshift-online and ``../install_config`` is not built for it either.